features/rspec_matcher.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Use reek_of mathcer
+  As a developer
+  In order to check code quality as part of my spec suite
+  I want to assert that my code doesn't smell
+
+  Scenario: Asserting a smelly file
+    Given the smelly file 'smelly.rb'
+    And a file "reek_spec.rb" with:
+    """
+    require 'reek/spec'
+
+    describe 'smelly.rb' do
+      it 'is fragrant' do
+        expect(Pathname.new('smelly.rb')).to reek
+      end
+    end
+    """
+    When I run `rspec reek_spec.rb`
+    Then it succeeds
+    And stdout should contain:
+    """
+    1 example, 0 failures
+    """
+
+  Scenario: Pick up configuration file
+    Given the smelly file 'smelly.rb'
+    And a configuration file 'full_mask.reek'
+    And a file "reek_spec.rb" with:
+    """
+    require 'reek/spec'
+
+    describe 'smelly.rb' do
+      it 'is fragrant' do
+        expect(Pathname.new('smelly.rb')).not_to reek
+      end
+    end
+    """
+    When I run `rspec reek_spec.rb`
+    Then it succeeds
+    And stdout should contain:
```

Comment:
The second scenario is almost identical with a couple of easy to miss differences. Would be great if we could make the differences more obvious.

Reply:
I've changed the scenarios around so I could extract more common parts. Let me know what you think.